odl/space/npy_ntuples.py

```diff
@@ -1387,6 +1387,26 @@ def adjoint(self):
         return MatVecOperator(self.matrix.conj().T,
                               domain=self.range, range=self.domain)
 
+    @property
+    def inverse(self):
+        """Inverse operator represented by the inverse matrix.
+
+        Taking the inverse causes sparse matrices to become dense and is
+        generally very heavy computationally. It is recommended to instead
+        use one of the solvers available in the solvers package.
+
+        Returns
+        -------
+        inverse : `MatVecOperator`
+        """
+        if self.matrix_issparse:
+            dense_matrix = self.matrix.toarray()
+        else:
+            dense_matrix = self.matrix
+
+        return MatVecOperator(np.linalg.inv(dense_matrix),
```

Comment:
Won't do that now, mostly because I highly doubt anyone will use it in a forseeable future. This operator is not intended to be used in "actual problems" but only in toy problems.